pom.xml

```diff
@@ -3,7 +3,7 @@
 
 	<groupId>io.github.classgraph</groupId>
 	<artifactId>classgraph</artifactId>
-	<version>4.4.1-SNAPSHOT</version>
+	<version>4.4.2-SNAPSHOT</version>
```

Comment:
This is the up-to-date pom from the origin. This won't appear in the diff when we make the real PR.